lib/datasets/e-stat-japan.rb

```diff
@@ -0,0 +1,324 @@
+# frozen_string_literal: true
+# TODO rename this file to e-stat-japan.rb
+require 'digest/md5'
+require 'net/http'
+require 'uri'
+require 'json'
+
+module Datasets
+  module EStatJapan
+    Record = Struct.new(:id, :name, :values)
+    # configuration injection
+    module Configuration
```

Comment:
How about `XXXable` like `Enumerable`?

```suggestion
    module Configurable
```

Reply:
Yes, I pushed https://github.com/red-data-tools/red-datasets/pull/90/commits/37817c1f1f8582ab5e1b20294c32ee49c1e3ae13